scripts/deno/bots.ts

```diff
@@ -44,7 +44,11 @@ const { QueryExecutionId } = await client.send(
   WHERE year = ${date.year}
 	  AND month = ${String(date.month).padStart(2, "0")}
 	  AND day = ${String(date.day).padStart(2, "0")}
-      AND request_user_agent NOT LIKE 'Guardian%'
+	  AND request_user_agent NOT LIKE 'Guardian/% CFNetwork%' 
+	  AND request_user_agent NOT LIKE 'GuardianNews/%'
+	  AND request_user_agent NOT LIKE 'Guardian\%20Editions/%'
```

Comment:
The user agents we use for iOS and Android in Athena requests are:

```
Android: GuardianNews/%
iOS: Guardian/%Darwin%
```

Chat thread [here](https://chat.google.com/room/AAAAehVq1H0/TA-ByiQPQwg/TA-ByiQPQwg?cls=10)